samples/sample_common/src/sample_utils.cpp

```diff
@@ -990,19 +995,19 @@ mfxStatus CSmplYUVWriter::WriteNextFrame(mfxFrameSurface1 *pSurface)
     case MFX_FOURCC_P010:
     case MFX_FOURCC_P210:
     {
-        mfxU32 height = pInfo.FourCC == MFX_FOURCC_P010 ? (mfxU32)pInfo.CropH / 2 : (mfxU32)pInfo.CropH;
+        mfxU32 height = pInfo.FourCC == MFX_FOURCC_P210 ? (mfxU32)pInfo.CropH : (mfxU32)pInfo.CropH / 2;
```

Comment:
what is the sense to change this line?

Reply:
It should be done for more flexible future adding of the new formats

Reply:
You just swapped if/else here. I don't understand why